tests/test_multichannel.py

```diff
@@ -59,6 +92,119 @@ def test_stft_multi(y_multi):
     assert not np.allclose(D0, D1)
 
 
+def test_onset_strength(y_multi):
+
+    # Verify that a stereo spectral flux onset strength envelope matches on
+    # each channel individually
+    y, sr = y_multi
+
+    S = librosa.stft(y)
+
+    D = librosa.onset.onset_strength(S=S)
+
+    D0 = librosa.onset.onset_strength(S=S[0])
+    D1 = librosa.onset.onset_strength(S=S[1])
+
+    # Check each channel
+    assert np.allclose(D[0], D0)
+    assert np.allclose(D[1], D1)
+
+    # Check that they're not both the same
+    assert not np.allclose(D0, D1)
+
+
+def test_tempogram(y_multi):
```

Comment:
this test should just use `s_multi` instead of y_multi`.

If `onset_envelope` is provided, then `y` will not be used in the tempogram calculations.